openmetadata-ui/src/main/resources/ui/src/components/IngestionStepper/IngestionStepper.css

```diff
@@ -96,3 +97,46 @@
   margin-top: 0.15rem;
   z-index: 1;
 }
+
+.ant-steps-label-vertical .ant-steps-item-tail {
+  padding: 3.5px 0px;
+}
+
+.ant-steps-item-finish
+  > .ant-steps-item-container
+  > .ant-steps-item-tail::after {
+  background-color: #d9ceee;
+  height: 2px;
+}
+
+.ant-steps-item-process
+  > .ant-steps-item-container
+  > .ant-steps-item-tail::after {
+  background-color: #d9ceee;
+  height: 2px;
+}
+
+.ant-steps-small .ant-steps-item-title {
+  font-size: 12px;
+  color: #7147e8;
+}
+
+.ant-steps-item-finish
+  > .ant-steps-item-container
+  > .ant-steps-item-content
```

Comment:
We want this style apply globally?
Then we should take it to `/style/component/step.less`

Reply:
The reason I gave local styling is becauseI have added the antd stepper for this component only. Should I replace it for all the places ?

Reply:
Yes let's have it across app

Reply:
Same component was being used everywhere so just moved the styling to /styles/component/step.less